### PR TITLE
Cnh/slice map 5176

### PR DESCRIPTION
**Pull request by kmer2016**

### CODE REVIEW

File: front/src/reducers/map/map.spec.ts [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+      longitude: 50,
+      zoom: 6.2,
+      bearing: 0,
+      pitch: 0,
+      padding: { top: 0, left: 0, bottom: 0, right: 0 },
+      width: 0,
+      height: 0,
+      transformRequest: transformMapRequest,
+    } as Partial<Viewport>;
+    store.dispatch(updateViewportAction(viewport));
```
Comment (SharglutDev) [inline front/src/reducers/map/map.spec.ts:50]: Shouldn't that (and the others) be harmonized with same structure as the one in Uriel's PR that has already been merged ?
```suggestion
    const action = updateViewportAction({
      latitude: 40,
      longitude: 50,
      zoom: 6.2,
      bearing: 0,
      pitch: 0,
      padding: { top: 0, left: 0, bottom: 0, right: 0 },
      width: 0,
      height: 0,
      transformRequest: transformMapRequest,
    } as Partial<Viewport>);
    store.dispatch(action);
```

File: front/src/reducers/map/map.spec.ts [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    expect(store.getState().map.viewport).toEqual(viewport);
```
Comment (SharglutDev) [inline front/src/reducers/map/map.spec.ts]: Same suggestion for harmonization with previous PR
```suggestion
   const mapState = store.getState().map
    expect(mapState.viewport).toEqual(viewport);
```

**Review by SharglutDev — COMMENTED** [review]
Lgtm. Map seems to be working fine, tested tests with yarn test. Good job !
Just a few suggestions :)

**Review by SharglutDev — APPROVED** [review]
Lgtm :)

**Review by Math-R — APPROVED** [review]

**Pull request merged**